server/helpers/emailTemplate.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * @description creates template for sending emails
+ * @returns object of temapltes for sending emails;
+ */
+const template = {
+    verification: {
+        from: 'no-reply@authorhaven.com',
+        subject: 'Email Verification',
+        text: 'Verify your Email at Authors Haven',
+        html: `
+        <h1 style="color: #6C54EC"> Welcome to Author's Haven</h1>
+        <p style="color:black">Thank you for signing up
+        for an author's haven account
+        please click on the button to verify your email address</p>
+        `
+    },
+    welcome: {
+        from: 'no-reply@authorhaven.com',
+        subject: 'Authors Haven accout created',
+        text: 'Thank you for creating an account at Author Haven',
+        html: `
+        <h1 style="color: #6C54EC"> Welcome to Author's Haven</h1>
+        Log in into your account to start contributing.
+         `
+    },
+    expiredToken: {
+        from: 'no-reply@authorhaven.com',
+        subject: 'Expired Token',
+        text: 'The token you attempt to verify your account with has expired',
+        html: `
+        <h1 style="color: #6C54EC"> Welcome to Author's Haven</h1>
+        You are receiving this mail because you made attempt to verify your
+        account with expired credentials kindly click the link
```

Comment:
This sentence might be better `You are receiving this mail because you made an attempt to verify your
account with expired credentials.
Kindly click the link below to verify your account.`